javascript/ql/src/semmle/javascript/security/dataflow/TaintedPathCustomizations.qll

```diff
@@ -355,6 +355,33 @@ module TaintedPath {
     }
   }
 
+  /**
+   * An expression of form `isInside(x, y)` or similar, where `isInside` is
+   * a library check for the relation between `x` and `y`.
+   */
+  class IsInsideCheckSanitizer extends DataFlow::BarrierGuardNode {
+    DataFlow::Node checked;
+
+    IsInsideCheckSanitizer() {
+      exists(string name, DataFlow::CallNode check |
+        name = "path-is-inside" or
```

Comment:
In line with @erik-krogh's comment, could you make `path-is-inside` only sanitize normalized absolute paths?

Reply:
I am waiting to see if Erik's PR against `path-is-inside` gets accepted.

Reply:
That PR seems to have entered the void. `path-is-inside` is now a weaker sanitizer.